src/main/resources/templates/swift/xcode/solution/${packageNameFolder}/SampleText.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class SampleText {
+    var text: String
+
+    init(text: String) {
+        self.text = text
+    }
+
+    convenience init() {
+        self.init(text: "")
+    }
+
```

Comment:
I would add a setText()-function

```suggestion
func setText(_ text: String) {
    self.text = text
}
```

Reply:
I don't see the need for this function, as it is just a simple class which takes in the text within the init(). 
And would also be problematic for the test code

Reply:
I thought we should have it because of making "text" private, but if it stays public it is not necessary.